scenarioo-client/test/e2e/specs/configure_label_colors.ts

```diff
@@ -15,16 +15,15 @@ useCase('Configure label colors')
 
         scenario('Create, edit and delete label configurations')
             .it(async () => {
-
                 // Given number of preconfigured colors (in config.xml for demo)
-                const numberOfPreconfiguredColors = 3;
+                const numberOfPreconfiguredColors: number = 3;
```

Comment:
type not needed here  - typescript recognises the type when assigning a value

Reply:
Thanks for the info, will keep that in mind